sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -90,6 +90,8 @@
 
     private final DatabaseManager mDatabaseManager;
 
+    private static final String FailedToReadCacheMessage = "Failed to read from cache.";
```

Comment:
nit: move this line before `private final DatabaseManager mDatabaseManager` to keep constant variables together.